### PR TITLE
updated js docs installation

### PR DESCRIPTION
**Pull request by Mik-TF**
* To answer [this issue](https://github.com/threefoldtech/info_grid/issues/138)
* Note that the tests on the grid_client are all failling. This is not an issue from the info_grid repository.
* The new Installation section provides the necessary information to install it and run the grid client docs.
   * Thus we know the installation is working. Fixing the tests and scripts will be needed on the tfgrid-sdk-ts side.
   * Once this is done, we can update the info_grid Grid Client documentation.

### CODE REVIEW

File: src/javascript/grid3_javascript_installation.md [hunk 1]
```diff
@@ -1,66 +1,125 @@
...
 
 ## Installation
 
-### External package
+### External Package
+
+To install the external package, simply run the following command:
 
 ```bash
 yarn add @threefold/grid_client
```
Comment (A-Harby) [inline src/javascript/grid3_javascript_installation.md:49]: should add node version [Node.js](https://nodejs.org/en) ^18

File: src/javascript/grid3_javascript_installation.md [hunk 2]
```diff
@@ -1,66 +1,125 @@
...
-### Local usage
+> Note: For the **qa**, **test** and **main** networks, please use @2.1.1 version.
+
+### Local Usage
+
+To use the Grid Client locally, clone the repository then install the Grid Client:
 
 - Clone the repository
+  - ```bash
+    git clone https://github.com/threefoldtech/tfgrid-sdk-ts
```
Comment (A-Harby) [inline src/javascript/grid3_javascript_installation.md:60]: Same here https://github.com/threefoldtech/tfgrid-sdk-ts#requirements

**Review by A-Harby — CHANGES_REQUESTED** [review]

File: src/javascript/grid3_javascript_installation.md [hunk 3]
```diff
@@ -1,66 +1,125 @@
...
+- may need to install libtool (**apt-get install libtool**)
 
-> [nvm](https://nvm.sh/) is the recommended way for installing node.
+> Note: [nvm](https://nvm.sh/) is the recommended way for installing node.
+
+To use the Grid Client, you will need the following on the TFGrid:
+
+- An account
+- A twin
+- Mnemonics
```
[the remark this answers is not in the document]
Comment (Mik-TF) [reply src/javascript/grid3_javascript_installation.md]: OK I set: a tfchain account and tft in the wallet

File: src/javascript/grid3_javascript_installation.md (hunk 1 above)
Comment (Mik-TF) [reply src/javascript/grid3_javascript_installation.md:48]: I added Node.js ^18 in the Prerequisites section

File: src/javascript/grid3_javascript_installation.md (hunk 2 above)
Comment (Mik-TF) [reply src/javascript/grid3_javascript_installation.md:59]: I added Node.js ^18 in the Prerequisites section

**Pull request merged**